packages/react-data-grid/src/masks/EventBus.js

```diff
@@ -0,0 +1,28 @@
+class EventBus {
```

Comment:
`Cell` component uses event bus to interact with the `InteractionMasks` component. The idea is to remove the local state from the `ReactDataGrid` component so that the whole tree is not rendered on every keystroke. 

- Click `Cell`
- Fire `SelectCell` event
- `InteractionMasks` is subscribed to the event and handle the selection logic